docs/design-docs/benchmarks.md

```diff
@@ -0,0 +1,274 @@
+---
+id: benchmarks
+---
+
+# Benchmarks Design
+
+## Goals
+
+The design of benchmarks has the following goals to achieve:
```

Comment:
Shall we change benchmarks to `benchmarks`? It looks like a term.

Reply:
Thanks, good point.